ktor-server/ktor-server-core/jvm/src/io/ktor/server/http/content/SinglePageApplication.kt

```diff
@@ -2,69 +2,53 @@
  * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
 
-package io.ktor.plugins.spa
+package io.ktor.server.http.content
 
 import io.ktor.http.*
-import io.ktor.server.application.*
-import io.ktor.server.http.content.*
 import io.ktor.server.request.*
-import io.ktor.server.response.*
 import io.ktor.server.routing.*
-import io.ktor.util.*
-import io.ktor.util.pipeline.*
 import java.io.*
 
 /**
- * A plugin that allows you to serve a single-page application
+ * Serves a single-page application
  *
- * A basic plugin configuration for the application served from the filesPath folder
+ * A basic configuration for the application served from the filesPath folder
  * with index.html as a default file:
  *
  * ```
- * install(SinglePageApplication) {
- *   filesPath = "application/project_path"
+ * application {
+ *     routing {
+ *        singlePageApplication {
+ *           filesPath = "application/project_path"
+ *         }
+ *     }
  * }
```

Comment:
Let's add a link to docs:

> You can learn more from [Serving single-page applications](https://ktor.io/docs/serving-spa.html).

Note that this link will return 404 until 2.0.0 is released.